GeneralsMD/Code/Tools/Autorun/CDCNTRL.CPP

```diff
@@ -1079,7 +1079,7 @@ bool WINAPI CDControlClass::Unlock_Logical_Volume (HANDLE vwin32, char drive)
  *   6/24/99 4:44PM MML : Created																*
  *==============================================================================================*/
 
-void Last_Error_Text ( LPTSTR szPrefix, HRESULT hr ) 
+void Last_Error_Text ( LPCTSTR szPrefix, HRESULT hr ) 
```

Comment:
Quoting Brian Kramer [on the MSDN forums](https://social.msdn.microsoft.com/forums/vstudio/en-US/1b994ff3-da28-46a8-90ef-6c7fab8dcd21/lpctstr#e4e05bbe-7cb8-4321-97de-fcf1c435952e)

    LPCTSTR = L‌ong P‌ointer to a C‌onst T‌CHAR STR‌ing 

    Here's the table:

       ```
 LPSTR = char*
        LPCSTR = const char*
        LPWSTR = wchar_t*
        LPCWSTR = const wchar_t*
        LPTSTR = char* or wchar_t* depending on _UNICODE
        LPCTSTR = const char* or const wchar_t* depending on _UNICODE
```